src/coreclr/src/debug/createdump/createdump.h

```diff
@@ -45,6 +45,10 @@ typedef int T_CONTEXT;
 #include <sys/user.h>
 #include <sys/wait.h>
 #include <sys/procfs.h>
+#ifdef __linux__
+#include <sys/uio.h>
+#define HAVE_READV
```

Comment:
Even though we are sure of such a feature in most modern kernels, why not test for it in https://github.com/dotnet/runtime/blob/875a54a112999004418ca369dfb0c31a7612c3cd/src/coreclr/src/pal/src/configure.cmake#L103
That way we have this consistent, users can build for other platforms with no hard-coded elements, and makes bring-up platforms easier.